src/server/index.ts

```diff
@@ -1,13 +1,9 @@
-import './initDotenv'
 import next from 'next'
 import log, { final } from './log'
 import createServer from './createServer'
-import Config from '../lib/Config'
+import env from './env'
```

Comment:
Per my other comment: https://github.com/bloomberg/coffeechat/pull/3/commits/92b506e67f2b647376cfb85836ea3e17ae7bc480#r624785942

I'm wondering if we could somehow export the properties. This is already pretty, good. I'm just wondering.